decidim-core/lib/decidim/core/test/factories.rb

```diff
@@ -809,6 +807,12 @@ def generate_localized_title
   factory :reminder_record, class: "Decidim::ReminderRecord" do
     reminder { create(:reminder) }
     remindable { build(:dummy_resource) }
+
+    Decidim::ReminderRecord::STATES.keys.each do |defined_state|
+      trait defined_state.to_sym do
```

Comment:
Isn't it already a symbol? I think we can remove `.to_sym`.
```suggestion
      trait defined_state do
```